testmodels/src/main/java/com/amplifyframework/testmodels/commentsblog/AmplifyModelProvider.java

```diff
@@ -34,7 +34,9 @@ public static AmplifyModelProvider getInstance() {
   @Override
    public Set<Class<? extends Model>> models() {
     final Set<Class<? extends Model>> modifiableSet = new HashSet<>(
-          Arrays.<Class<? extends Model>>asList(Blog.class, Post.class, Comment.class, PostAuthorJoin.class, Author.class, BlogOwner.class)
+            Arrays.<Class<? extends Model>>asList(Blog.class, Post.class, Comment.class, PostAuthorJoin.class, Author.class, BlogOwner.class)
+            //Arrays.<Class<? extends Model>>asList(Blog.class, Post.class, Comment.class, PostAuthorJoin.class,
+            // Author.class, BlogOwner.class, BlogOwnerWithCustomPK.class)
```

Comment:
Should `BlogOwnerWithCustomPK.class` be included in the list?

Reply:
Removed will add the integration test in another PR.